pkg/common/util/stringutil/stringutil.go

```diff
@@ -18,6 +18,8 @@
 package stringutil
 
 import (
+	perrors "github.com/pkg/errors"
```

Comment:
use ` imports-formatter -path . -module github.com/apache/dubbo-go-pixiu -bl false` to auto group imports line